README.md

```diff
@@ -82,7 +85,17 @@ latest YY.MM (year.month) of [Triton release](https://github.com/triton-inferenc
 
 ```
 # YY.MM is the version of Triton.
-export TRITON_CONTAINER_VERSION=<YY.MM>
+# Get latest VLLM RELEASED VERSION from https://github.com/triton-inference-server/vllm_backend/releases
+TAG=$(curl https://api.github.com/repos/triton-inference-server/vllm_backend/releases/latest | grep -i "tag_name" | awk -F '"' '{print $4}')
+export TRITON_CONTAINER_VERSION=${TAG#v} # example: 24.06
+echo "TRITON_CONTAINER_VERSION = ${TRITON_CONTAINER_VERSION}"
+
+# Get latest VLLM RELEASED VERSION from https://github.com/vllm-project/vllm/releases
+TAG=$(curl https://api.github.com/repos/vllm-project/vllm/releases/latest | grep -i "tag_name" | awk -F '"' '{print $4}')
+export VLLM_VERSION=${TAG#v} # example: 0.5.3.post1
+echo "VLLM_VERSION = ${VLLM_VERSION}"
+
+git clone -b r${TRITON_CONTAINER_VERSION} https://github.com/triton-inference-server/server.git
```

Comment:
great suggestion, let's also add `cd server` here and add `\` symbol to `build.py` command